### PR TITLE
env: use prjoxide from conda

### PR DESCRIPTION
**Pull request by acomodi**
Signed-off-by: Alessandro Comodi <acomodi@antmicro.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -23,32 +23,11 @@ include third_party/make-env/conda.mk
...
 		popd
-
-	pushd third_party/prjoxide/libprjoxide && \
-		( curl --proto '=https' -sSf https://sh.rustup.rs | sh -s -- -y ) && \
-		source ${HOME}/.cargo/env && \
-		cargo install --path prjoxide --all-features && \
-		popd
 	# Build test suite
-	@$(IN_CONDA_ENV) mkdir -p build && cd build && cmake $(CMAKE_FLAGS) -DPRJOXIDE=${HOME}/.cargo/bin/prjoxide ..
+	@$(IN_CONDA_ENV) mkdir -p build && cd build && cmake $(CMAKE_FLAGS)
```
Comment (gatecat) [inline Makefile:30]: looks like the `..` might have been deleted here as well by mistake?

Comment (acomodi) [reply Makefile]: Yep, thanks!

**Comment by gatecat** [comment]
I think you can also remove the `prjoxide` git submodule in this patch?

**Comment by acomodi** [comment]
> I think you can also remove the prjoxide git submodule in this patch?

Indeed, removed.

**Review by gatecat — APPROVED** [review]
LGTM

**Pull request merged**